docs/src/index.md

```diff
@@ -1,6 +1,6 @@
-# NonlinearSolve.jl: High-Performance Unified Nonlinear Solvers
```

Comment:
it should keep its title

Reply:
But then it will look similar to this in the dropdown menu.
![Capture](https://user-images.githubusercontent.com/11204473/182279243-fc1e4aa7-1b5b-4321-b222-04b36053f18d.PNG)
which is a bit ugly in my opinion.
So how do you want it? **Overview or home** on the dropdown and the actual name of the package on the corresponding page?

Reply:
That doesn't come from the top of the page, it comes from the title description here: https://github.com/SciML/Optimization.jl/blob/master/docs/pages.jl#L2. But it's actually rather important for SEO reasons that the top file says "Optimization.jl" instead of "Home", otherwise Google will never pick up the page and put it to the top of the ranking as "Optimization.jl". So we really should be fixing the other ones to be more like DifferentialEquations.jl in that respect to fix how it interacts with search engines. Though I agree it would be nice to decouple that from what ends up in the sidebar.

Reply:
Ok hopefully this can serve as a template for the other packages to be  made uniform.